.husky/pre-commit

```diff
@@ -1,4 +1,41 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+# Pack CircleCI config if any files in .circleci/src were changed
+if git diff --cached --name-only | grep -q "^\.circleci/src/"; then
+  if ! which circleci > /dev/null; then
+    echo "❌ Warning: CircleCI CLI not found! Install the CircleCi CLI to edit the CircleCI configuration."
+    exit 1
+  fi
```

Comment:
Verified this works 
<img width="693" height="146" alt="Screenshot 2025-09-17 at 10 41 58 AM" src="https://github.com/user-attachments/assets/34a779f9-fc30-4092-82f0-7dc423d347e0" />